crates/rpc/rpc/src/debug.rs

```diff
@@ -155,7 +155,11 @@ where
         let block = self.client.block(block_id).to_rpc_result()?;
 
         let mut res = Vec::new();
-        if let Some(block) = block {
+        if let Some(mut block) = block {
+            // In RPC withdrawals are always present
+            if block.withdrawals.is_none() {
+                block.withdrawals = Some(vec![]);
+            }
```

Comment:
I see. A bit meh footgun here. Should it be `withdrawals: Vec<_>`, instead of `withdrawals: Option<Vec<_>>`?

Reply:
Wouldn't doing that break the RLP impl because of how we use `rlp(trailing)` and the option?

Reply:
this more of a quirk of geth, wich does not have options so empty is the same as missing so missing is also encoded as trailing `c0`